README.md

```diff
@@ -1,2 +1,173 @@
+
+
+**[Conference-page]** is a project where I did a conference page based on guidelines provides by microverse for the final project of the first module. I implemented the home and about pages with desktop and mobile version using HTML and CSS and with functionality using JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML5</li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li>CSS</li>
+  </ul>
+</details>
+<details>
+  <summary>JAVASCRIPT</summary>
+  <ul>
+    <li>JAVASCRIPT</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Responsive]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [[Github-pages]](https://alejandro-bernal-m.github.io/Conference-page/)
+- [walkthrough](https://www.loom.com/share/ef90bfe051e843429fb5369c6c89996f)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+you can use this code as a base for your own project.
+
+### Prerequisites
+
+In order to run this project you need:
+CSS/HTML/JavaScript
+
+
+### Setup
+
+Just copy and paste what you need but be sure to understand.
+
+
+### Install
+
+There is no need to install anything
+
+### Usage
+
+Just copy and paste what you need but be sure to understand.
+
+
+
+### Run tests
+
+Open it in your browser.
+
+
+### Deployment
+
+Just open it in your browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [Alejandro](https://github.com/Alejandro-Bernal-M)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[no-future-features]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
+This project is based on the project of the author [cindy-shin-on-behance](https://www.behance.net/adagio07)
+This is the [original-project](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```

Comment:
- [ ] I would kindly suggest you work on the MIT license link as it is currently broken and be sure it is working perfectly fine.

The image 👇 

![mit](https://user-images.githubusercontent.com/78634841/217394514-4eecbb29-30b3-48bb-ab69-c654b51dd5c6.png)